source/_integrations/http.markdown

```diff
@@ -154,6 +157,9 @@ Please note, that sources from `trusted_networks` won't be banned automatically.
 
 </div>
 
+If you are using a load balancer (e.g. Nginx, Traefik etc) with Docker then all requests will appear to come from the IP of your load balancer and therefore IP block all clients. To get the real IP you must set `use_x_forwarded_for` to true and add Dockers IP range to trusted_proxies (the default Docker IP range is 172.17.0.0/16 but it will use other ranges if this is already taken on your network).
```

Comment:
This is partially true. I would not recommend anybody to use a CIDR for this, because it simply opens up the attack surface. In the setting of Home Assistant assuming that users using a proxy, are probably just using a single proxy and thus IP (I highly doubt a cluster of proxies).

Reply:
So the problem I was trying to solve is that the reverse proxy / load balancer might have a different IP address each time Docker starts. This is to my knowledge the default situation with HASS.io

Is there a better solution to this issue?